crates/sui/src/ethereum_client_commands.rs

```diff
@@ -0,0 +1,85 @@
+        /// The DWallet ID.
+        #[clap(long)]
+        dwallet_id: ObjectID,
+        /// Gas object for gas payment.
+        #[clap(long)]
+        gas: Option<ObjectID>,
+        /// Gas budget for this call
+        #[clap(long)]
+        gas_budget: u64,
+        /// Instead of executing the transaction, serialize the bcs bytes of the unsigned transaction data
+        /// (TransactionData) using base64 encoding, and print out the string.
+        #[clap(long, required = false)]
+        serialize_unsigned_transaction: bool,
+        /// Instead of executing the transaction, serialize the bcs bytes of the signed transaction data
+        /// (SenderSignedData) using base64 encoding, and print out the string.
+        #[clap(long, required = false)]
+        serialize_signed_transaction: bool,
+    },
+
+    /// Connect dWallet to be controlled by Eth contract.
+    #[command(name = "connect-eth-dwallet")]
+    CreateEthDwallet {
+        /// The ObjectID of the dWallet *cap*ability.
+        #[clap(long)]
+        dwallet_cap_id: ObjectID,
+        /// The address of the contract.
+        #[clap(long)]
+        contract_address: String,
+        /// The slot of the Data map that holds approved transactions in eth smart contract.
+        #[clap(long)]
+        contract_approved_tx_slot: u64,
+        /// The address of the gas object for gas payment.
+        #[clap(long)]
+        gas: Option<ObjectID>,
+        /// Gas budget for this call
+        #[clap(long)]
+        gas_budget: u64,
+        /// Instead of executing the transaction, serialize the bcs bytes of the unsigned transaction data
+        /// (TransactionData) using base64 encoding, and print out the string.
+        #[clap(long, required = false)]
+        serialize_unsigned_transaction: bool,
+        /// Instead of executing the transaction, serialize the bcs bytes of the signed transaction data
+        /// (SenderSignedData) using base64 encoding, and print out the string.
+        #[clap(long, required = false)]
+        serialize_signed_transaction: bool,
+    },
+
+    /// Initiate the LatestEthereumState struct in the DWallet module.
+    #[command(name = "init-eth-state")]
+    InitEthState {
+        /// A Trusted checkpoint to initialize the state.
+        #[clap(long)]
+        checkpoint: String,
+        /// The address of the gas object for gas payment.
+        #[clap(long)]
+        gas: Option<ObjectID>,
+        /// Gas budget for this call.
+        #[clap(long)]
+        gas_budget: u64,
+        /// Instead of executing the transaction, serialize the bcs bytes of the unsigned transaction data
+        /// (TransactionData) using base64 encoding, and print out the string.
+        #[clap(long, required = false)]
+        serialize_unsigned_transaction: bool,
+        /// Instead of executing the transaction, serialize the bcs bytes of the signed transaction data
+        /// (SenderSignedData) using base64 encoding, and print out the string.
```

Comment:
```suggestion
        /// Instead of executing the transaction,
        /// serialize the bcs bytes of the signed transaction data
        /// (SenderSignedData) using base64 encoding, and print out the string.
```